### PR TITLE
address kconfig post merge issues and document api

### PR DESCRIPTION
**Pull request by rafaeldtinoco**

### CODE REVIEW

File: docs/helpers.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+#### KernelConfig (Bonus)
+
+Through the KernelConfig API is also possible to 'add' custom
+KernelConfigOption's variables that one might need for their eBPF's program
+execution. Example:
+
+```go
+	const (
+		CONFIG_ARCH_HAS_SYSCALL_WRAPPER helpers.KernelConfigOption = iota + 1000
```
[the remark this answers is not in the document]
Comment (rafaeldtinoco) [reply docs/helpers.md:109]: Thats weird, my local repo had this already. Maybe I fixed in another branch and forgot to push, thanks.

File: helpers/kernel_config.go [hunk 2]
```diff
@@ -231,12 +232,21 @@ func (k *KernelConfig) GetKernelConfigFilePath() string {
...
-	KernelConfigKeyIDToString[key] = value
-	KernelConfigKeyStringToID[value] = key
+	kernelConfigKeyIDToString[key] = value
+	kernelConfigKeyStringToID[value] = key
+
+	return nil
+}
 
+// LoadKernelConfig will (re)read kconfig file (likely after AddCustomKernelConfig was called)
+func (k *KernelConfig) LoadKernelConfig() error {
```
[the remark this answers is not in the document]
Comment (rafaeldtinoco) [reply helpers/kernel_config.go:249]: tracee/tracee.go:	if err = t.config.KernelConfig.LoadKernelConfig(); err != nil { // invalid kconfig file: assume values then

[the remark this answers is not in the document]
Comment (rafaeldtinoco) [reply helpers/kernel_config.go:249]: https://github.com/aquasecurity/tracee/pull/999

File: helpers/kernel_config.go [hunk 3]
```diff
@@ -87,9 +87,10 @@ const (
 	CONFIG_BPF_LSM
 	CONFIG_BPF_PRELOAD
 	CONFIG_BPF_PRELOAD_UMD
+	CUSTOM_OPTION_MARK KernelConfigOption = 1000
```
Comment (yanivagman) [inline helpers/kernel_config.go:90]: not a must, but maybe CUSTOM_OPTION_START would be a better fit?

**Review by yanivagman — DISMISSED** [review]
LGTM

File: helpers/kernel_config.go (hunk 3 above)
Comment (rafaeldtinoco) [reply helpers/kernel_config.go:90]: I currently don't have a way to push without a review. I'm changing it and will kindly ask you @yanivagman to approve this again (BTW, should we change this policy for nit fixes ? Do we always want to enforce nit fixes push with a big turn around of 1 or 2 days, until the review is done again ?)

**Review by yanivagman — APPROVED** [review]

**Pull request merged**